app/components/primer/spinner_component.rb

```diff
@@ -31,6 +31,10 @@ def initialize(size: DEFAULT_SIZE, **system_arguments)
       @system_arguments = system_arguments
       @system_arguments[:tag] = :svg
       @system_arguments[:style] ||= DEFAULT_STYLE
+      @system_arguments[:classes] = class_names(
+        @system_arguments[:classes],
+        "anim-rotate"
+      )
```

Comment:
```suggestion
      @system_arguments[:animation] = :rotate
```

this will add `anim-rotate` for you